crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -176,10 +177,12 @@ async fn test_transfer_sui_insufficient_gas() {
     let authority_state = init_state().await;
     authority_state.insert_genesis_object(gas_object).await;
 
-    let kind = TransactionKind::Single(SingleTransactionKind::TransferSui(TransferSui {
-        recipient,
-        amount: None,
-    }));
+    let pt = {
+        let mut builder = ProgrammableTransactionBuilder::new();
+        builder.transfer_sui(recipient, None);
+        builder.finish()
+    };
```

Comment:
Just to make sure, there's no blocker for removing `transfer_sui` given that we have `pay_sui` right? cc @gegaowp

Reply:
yes, deprecation of `transfer_sui` was held only b/c we did not want to do it twice, to pay_sui and then to PT